src/pages/Components/Radio/previews/radio-disabled-state.preview.html

```diff
@@ -0,0 +1,5 @@
+<ul class="example">
+    <li class="example__list-item">
+        <input id="radio-06" disabled="true" type="radio" class="mc-radio is-focus" name="example" />
```

Comment:
is-focus is wanted here ?

Reply:
@gael-boyenval no it's not necessary here